data/blog/electron-3-0.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Electron 3.0.0
+author: codebytere
+date: '2018-09-20'
+---
+
+The Electron team is excited to annouce that the first stable release of Electron `v3.0` is now
```

Comment:
* s/annouce/announce/

 * nit: `v3.0` isn't a version number. Maybe "first stable release of Electron 3"?